index.js

```diff
@@ -1,59 +1,81 @@
-const arr = JSON.parse(localStorage.getItem('array')) || [];
+class Books {
+  constructor(title, author) {
+    this.title = title;
+    this.author = author;
+  }
 
-function add() {
-  const title = document.querySelector('#title');
-  const author = document.querySelector('#author');
-  const value1 = title.value.trim();
-  const value2 = author.value.trim();
-  if ((value1.length !== 0) && (value2 !== 0)) {
-    const book = {
-      title: value1,
-      author: value2,
-    };
-    arr.push(book);
-    localStorage.setItem('array', JSON.stringify(arr));
+  add() {
+    const booklist = JSON.parse(localStorage.getItem('array')) || [];
```

Comment:
[OPTIONAL] Good job on implementing the requested change, but as previously hinted by the reviewer, it is good to put this in the constructor to avoid repeating yourself.

```
 constructor(title, author) {
    this.title = title;
    this.author = author;
    this.data = JSON.parse(localStorage.getItem('array')) || [];
  }
```